web/src/activity.ts

```diff
@@ -1,22 +1,41 @@
 import $ from "jquery";
+import assert from "minimalistic-assert";
+import {z} from "zod";
 
 import * as channel from "./channel";
 import {page_params} from "./page_params";
 import * as presence from "./presence";
 import * as watchdog from "./watchdog";
 
+const post_presence_response_schema = z.object({
+    msg: z.string(),
+    result: z.string(),
+    server_timestamp: z.number().optional(),
```

Comment:
Isn't `server_timestamp` always present?

Reply:
Ahh, I think that's a migration bug in 13d1241eed925e0b6247911e606d8aaa661bdfd5; @andersk FYI.

Original:
```
-    function get_full_presence_list_update() {
-        send_presence_to_server(true);
-    }

```

New version:
```
+    function get_full_presence_list_update() {
+        activity.send_presence_to_server(redraw);
+    }
```

Reply:
Ahh, no, I just missed that it was changed from a boolean to a function. Sorry for the confusion.